src/notebooks/filesystem-tutorial.ipynb

```diff
@@ -223,9 +232,10 @@
        "      <td>604863</td>\n",
        "      <td>Mary Archer</td>\n",
        "      <td>Mary.Archer@studentgps.org</td>\n",
-       "      <td>Archived</td>\n",
-       "      <td>8/20/2020 12:34</td>\n",
-       "      <td>9/18/2020 12:34</td>\n",
+       "      <td>2020-08-20 12:34:56</td>\n",
+       "      <td>2020-09-18 12:34:00</td>\n",
+       "      <td>NaT</td>\n",
```

Comment:
`NaT`? That's a new one. I'm not concerned by it - nothing we can do anyway. Just commenting on the funny way of indicating "no value"